resources/views/filament/forms/components/campaigns/actions/proactive-alert.blade.php

```diff
@@ -52,7 +55,7 @@
         </div>
         <div class="flex flex-col pt-3">
             <dt class="mb-1 text-sm text-gray-500 dark:text-gray-400">Status</dt>
-            <dd class="text-sm font-semibold">{{ $action['status'] }}</dd>
+            <dd class="text-sm font-semibold">{{ $alertStatus?->name }}</dd>
```

Comment:
This null safe operator is not needed. You are using a `findOrFail` above, so if the value was not retrieved out of the database then an exception would already have been thrown.